Sdtrig.tex

```diff
@@ -131,7 +131,9 @@ \section{Priority}
 \end{table}
 
 When multiple triggers in the same priority fire at once, \FcsrMcontrolHit (if
-implemented) is set for all of them. If one of these triggers has
+implemented) is set for all of them.  If more than one of these triggers has
+\FcsrMcontrolSixAction=0 then {\tt tval} is updated in accordance with one of
+them (which one is \unspecified).  If one of these triggers has
```

Comment:
```suggestion
them, but which one is \unspecified.  If one of these triggers has
```

Reply:
I feel parens are for explanation etc, while this is an important part of the spec.

Reply:
Yes, that makes sense.